builder/node/node_5.7.0/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# published as clever/drone-node:5.7.0
+FROM bradrydzewski/base
+
+WORKDIR /home/ubuntu
+USER ubuntu
+ADD nodejs.sh /etc/drone.d/
+
+RUN /bin/bash -c ". /home/ubuntu/nvm/nvm.sh && nvm install v5.7.0 && npm install npm@2.7.0 -g"
```

Comment:
Yeah, I noticed that it was rolling back too! Maybe I was copying from a
project that was incorrect. (I think I looked at launchpad or oauth)

Would you recommend the bundled version of npm, then, and no separate npm
install step?

On Wed, Apr 13, 2016, 18:05 jefff <notifications@github.com> wrote:

> In builder/node/node_5.7.0/Dockerfile
> <https://github.com/Clever/images/pull/11#discussion_r59651465>:
>
> > @@ -0,0 +1,8 @@
> > +# published as clever/drone-node:5.7.0
> > +FROM bradrydzewski/base
> > +
> > +WORKDIR /home/ubuntu
> > +USER ubuntu
> > +ADD nodejs.sh /etc/drone.d/
> > +
> > +RUN /bin/bash -c ". /home/ubuntu/nvm/nvm.sh && nvm install v5.7.0 && npm install npm@2.7.0 -g"
>
> Why npm 2.7.0? I think 3 offers advantages so while we're upgrading to
> node 5 it seems like a good time to get this out of way too, since it's the
> default version included with node 5.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Clever/images/pull/11/files/0c88ad405f96f94b0b2f4d9a01a903c04134794e#r59651465>
>